terraform/groups/ewf/main.tf

```diff
@@ -147,3 +147,14 @@ resource "aws_vpc_security_group_ingress_rule" "iboss_443" {
   to_port     = 443
   ip_protocol = "tcp"
 }
+
+resource "aws_vpc_security_group_ingress_rule" "allow_from_shared_services_management" {
+  count = contains(local.test_environments, var.environment) ? 1 : 0
```

Comment:
This can be simplified by utilising the existing `test_access_enable` boolean var. Currently it's only set in the `heritage-development-eu-west-2` but as test access is required in `staging` too, the profile can be updated to define the var as `true`.